0009-S-sdk-resiliency.md

```diff
@@ -66,12 +66,11 @@ Cons:
 ### Design
 
 * `DAPR_API_MAX_RETRIES` defines the maximum number of retries, SDKs can determine which strategy will be implemented (linear, exponential backoff, etc). `0` is the default value and means no retry. `-1` or any negative value means infinite retries.
```

Comment:
Would this be improved by also adding a env var the strategy? This can obviously be unset for a default but I'd like to see the option of a customisable strategy or at least definable perhaps

Reply:
I personally think we should always use exponential backoff and call it a day. I don't think we should overcomplicate the SDK implementation and configuration options.

Maybe instead we could agree on the default exponential backoff parameters.

Those parameters can be made configurable by individual SDKs if desired, but I don't think this proposal (which was basically already implemented) should be expanded.